api2/pkg/parties/parties/store.go

```diff
@@ -7,17 +7,17 @@ import (
 )
 
 type Store struct {
-	collection *mongo.Collection
+	Collection *mongo.Collection
```

Comment:
I don't think this should be exported... 
```suggestion
	collection *mongo.Collection
```

Reply:
I need it to be exported to be able to wrap it in the relationshipparties store